### DISCUSSION TITLE
For what choose client when creating a client

### DISCUSSION THREAD

**Discussion by matthias-ronge** [kitodo/kitodo-production#4406] [Q&A] [answered]
When creating a client, a selection box for clients is displayed. What is this good for?

![Screenshot](https://user-images.githubusercontent.com/3040657/82319310-9bd71480-99d1-11ea-98a0-15f913e38420.jpg)

**Comment by solth** [comment]
To copy all roles assigned to that client to the new client.

**Comment by matthias-ronge** [comment]
This is not intuitive, but it is definitely a function that is needed. Can I still copy the roles afterwards? I have already created client, with projects etc.

**Comment by solth** [comment]
No, I think we have no function to copy roles between already existing clients.

**Comment by Kathrin-Huber** [comment, marked as answer]
Of course you can!
Just edit the existing client, then you get the same dropdown to copy roles.

**Comment by solth** [comment]
Oops :trollface:

**Comment by solth** [comment]
@matthias-ronge if this is solved, why don't you close the issue?

**Comment by matthias-ronge** [comment]
My idea was to leave <kbd>question</kbd> tickets open until the problem and the solution were included in some central documentation.
